react/ProductSummaryCustom.tsx

```diff
@@ -204,6 +204,7 @@ interface Props {
    */
   position?: number
   classes?: CssHandlesTypes.CustomClasses<typeof CSS_HANDLES>
+  target?: string
```

Comment:
What do you think about this suggestion??
```suggestion
  target?: Link['target']
```